proposals/partial-properties.md

```diff
@@ -0,0 +1,142 @@
+# Partial properties
+https://github.com/dotnet/csharplang/issues/6420
+
+### Grammar
+
+The *property_declaration* grammar [(§14.7.1)](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/classes.md#1471-general) is updated as follows:
+
+```diff
+property_declaration
+-    : attributes? property_modifier* type member_name property_body
++    : attributes? property_modifier* 'partial'? type member_name property_body
+    ;  
+```
+
+**Remarks**: This is similar to how *method_header* [(§14.6.1)](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/classes.md#1461-general) and *class_declaration* [(§14.2.1)](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/classes.md#1421-general) are specified. (Note that [Issue #946](https://github.com/dotnet/csharplang/issues/946) proposes to relax the ordering requirement, and would probably apply to all declarations which allow the `partial` modifier.)
```

Comment:
FWIW, it looks like some of those links are out-of-date. But I think it's unavoidable as the standard evolves... :-/

Reply:
Yes, but we can probably reduce the breaks by preferring references to `standard-` instead of `draft-` branches in the future. I'll go ahead and update the references now.